src/components/Button.module.scss

```
@@ -0,0 +1,10 @@
+@import '../styles/style.scss';
```

```suggestion
@import 'src/styles/style.scss';
```
이렇게 절대 경로로 하면 매번 상대경로로 파악하지 않고 동일한 코드를 복붙..ㅎ 할 수 있어서 편할 것 같아요! 이것도 저스트 제안이니.. 패스

---

ㅋㅋㅋㅋㅋ이따 수정할 때 저 부분도 반영할게요